src/navbar/navbar.component.js

```diff
@@ -82,6 +82,11 @@ function menuItems() {
 					Vue.js
 				</a>
 			</li>
+			<li>
+				<a href="#/svelte">
```

Comment:
Shouldn't there also be a link in https://github.com/CanopyTax/single-spa-examples/blob/master/src/home/root.component.js#L64 for svelte?